server/test/red-flag.spec.js

```diff
@@ -28,6 +32,8 @@ describe('Ride-My-Way', () => {
       const initialRedFlags = db.length;
       const response = await server.post('/api/v1/red-flags').send(newRedFlag);
       expect(response.body.status).to.equal(201);
+      expect(response.body.data).to.be.an('array');
+      expect(response.body.data[0].message).to.eql('Red-Flag created successfully');
```

Comment:
Hi, I can see you are testing to check if the message returned is correct. I think you should also check that the data returned is correct and same as what was sent to the post API.